packages/datagrid/src/components/DataGrid/DataGrid.test.js

```diff
@@ -254,7 +254,7 @@ describe('#DataGrid', () => {
 	});
 
 	it('should render one Skeleton is InProgress', () => {
-		const wrapper = shallow(<DataGrid loading getComponent={getComponent} />);
+		const wrapper = shallow(<DataGrid loading />);
```

Comment:
`getComponent ` was useless for the test, with `loading` it returns before its usage